### ISSUE TITLE
Cannot find type 'NLContextualEmbedding' in scope

### ISSUE THREAD

**Issue by naticio** [huggingface/swift-coreml-diffusers#50]
I updated the common.xconfig file but still get this error

Cannot find type 'NLContextualEmbedding' in scope

for this line

let embeddingModel: NLContextualEmbedding

**Comment by dolmere** [comment]
I literally JUST figured this out as your message came in. The latest code is built using Xcode 15 beta. NLContextualEmbedding is a new API requiring the the beta targets it seems. I downloaded the beta from https://developer.apple.com/xcode/ and it builds fine now.

**Comment by naticio** [comment]
yes! sorry I figured it out as well, currently waiting for Xcode 15 to finish downloading, thanks!
